docusaurus/docs/guides/deployments/10-linux/20-router/10-deploy.mdx

```diff
@@ -0,0 +1,112 @@
+
+Explore configuration option variables by running `ziti create config environment --help`. Any of these may be set in `/opt/openziti/etc/router/bootstrap.env` to influence the generated configuration.
+
+Disable generating a configuration by setting `ZITI_BOOTSTRAP_CONFIG=false` in `/opt/openziti/etc/router/service.env`.
+
+### Bootstrapping the Enrollment
+
+You must provide an enrollment token. Obtain the token by administratively creating the router.
+
+```text title="Create a router"
+ziti edge create edge-router "AcmeRouter1" \
+    --tunneler-enabled \
+    --jwt-output-file path/to/token.jwt
+```
+
+[Learn more about managing routers with the CLI](/guides/deployments/10-linux/20-router/40-cli-mgmt.md).
+
+The systemd service looks for the token in `/opt/openziti/etc/router/.token` if env var `ZITI_ENROLL_TOKEN` is empty. The file must be readable by root (not others).
+
+`ZITI_ENROLL_TOKEN` may be defined in `/opt/openziti/etc/router/bootstrap.env`.
+
+The router will enroll with the controller during the first startup. The one-time enrollment token is consumed during the enrollment process and a private key is generated in the router's working directory.
+
+Disable bootstrapping enrollment by setting `ZITI_BOOTSTRAP_ENROLLMENT=false` in `/opt/openziti/etc/router/service.env`.
+
+## Firewall
+
+The router listens on a single configurable TCP port: `3022/tcp`. Ziti clients use ALPN to negotiate a connection to the edge (`ziti-edge`), fabric links (`ziti-link`), or health-check APIs (`h2`, `http/1.1`).
+
+You may set `ZITI_ROUTER_PORT` in `/opt/openziti/etc/router/bootstrap.env` to bootstrap with a different port.
+
+Clients "learn" the router's address and port when they poll the controller for authorized routers, so it is not necessary to re-enroll or re-create the client if the router's address or port changes. Simply change the config, bounce the router, and it will begin advertising the new address and port. This works because the system service and Docker container both auto-renew their server certificate every startup, and the certificate's DNS subject alternative name is set in the router's config.yml file. If you set `ZITI_AUTO_RENEW_CERTS=false` then it's necessary to administratively re-create the router or run at least once with the `--extend` flag.
+
+## Agent
+
+The router provides an IPC agent for administration. The agent listens on a Unix domain socket inside the filesystem namespace of the router service. Here's an example for querying the router's agent for statistics.
+
+```text
+systemctl show -p MainPID --value ziti-router.service \
+| xargs -rIPID sudo nsenter --target PID --mount -- \
+    ziti agent stats
+```
+
+```buttonless title="Output"
+goroutines: 38
+OS threads: 20
+GOMAXPROCS: 16
+num CPU: 16
+```
+
+## Logging
+
+<CliLogFormats/>
```

Comment:
All the same comments from the controller here. I don't think we can resuse this and i'd like to see treatment for log shipping added. I also want to tell them "you will use json and like it" because that's what the nfconsole does for good reasons.